lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb

```diff
@@ -23,8 +23,14 @@ class Console::CommandDispatcher::Stdapi::Fs
 
   CHECKSUM_ALGORITHMS = %w{ md5 sha1 }
   private_constant :CHECKSUM_ALGORITHMS
-  PATH_EXPAND_REGEX = /\%(\w*)\%/
-  private_constant :PATH_EXPAND_REGEX
+
+  def path_expand_regex
```

Comment:
This method is replacing a constant which used to be private, so the new method should probably also be private.

Reply:
Fixed in 32bf9f60d126ddb11a925cc1ee75597810076b70